docs/user_guide/capsules/index.md

```diff
@@ -1,43 +1,53 @@
+
+However, the goal if capsules is to have an endpoint through which all client communication can be routed. See [Capsule Isolation](/docs/{{ site.version }}/user_guide/capsules/isolation.html) for more information.
+
+With a fully configured Capsule, clients:
+    * Get network configuration from the DHCP server
+    * PXE boot from the TFTP server
+    * Retrieve kickstart templates from the Capsule
+    * Provision from kickstart trees that have been synced to the Capsule
+    * Configure itself using the Puppet Master
+    * Recieve content
 
 ## Deployment
 
-Most deployments would have a single Katello server with multiple Capsules attached.  You may want to deploy a Capsule into each data center, or into each region where there are systems you wish to manage.  Capsules can also be used to scale the number of systems attached to a single Katello server.  
+In the simplest use case, a user may only want to use the default capsule embedded into the Katello.  Larger deployments would have a single Katello server with multiple Capsules attached.  You may want to deploy a Capsule into each data center, or other remote location you wish to manage.  Capsules can also be used to scale the number of systems attached to a single Katello server.
 
-For example, if you have two small data centers hosting 1000 systems and one very large data center hosting 10,000 systems you would likely want to deploy one Capsule server into each of your small data centers and then 2 or more Capsules into your large data center.
+For example, if you have two small data centers hosting 1000 systems and one very large data center hosting 10,000 systems you would likely want to deploy one Capsule server into each of your small data centers and then two or more Capsules into your large data center.
 
 ## How do Capsules differ from Smart Proxies?
 
-If you are familiar with a Foreman Smart Proxy a Capsule is very similar but adds the content mirroring capabilities of yum and puppet content.  You may see the term Capsule and Smart Proxy used interchangeably.
+A Katello Capsule is a Foreman Proxy with the addition of Content-related services.
 
 ## What is the default Capsule?
 
-On every Katello server a Capsule is installed by default with only the Puppet Master feature enabled.  This feature is required to be enabled for proper use of Puppet content within Katello.
-
-There is no 'content mirroring' capability of the default Capsule since Katello already hosts the content locally.
+On every Katello server a Capsule is installed by default with Pulp and Puppetmasters enabled.  This feature is required to be enabled for proper use of Puppet content within Katello. There is no 'content mirroring' capability of the default Capsule since Katello already hosts the content locally, so the Pulp feature on the Capsule is special - it is the Pulp master.
```

Comment:
After reading this, I think maybe we should capitalize the 'Default Capsule' where used since it is a thing that we reference quite often.